pubsub/jetstream/jetstream.go

```diff
@@ -166,3 +176,16 @@ func (js *jetstreamPubSub) Close() error {
 
 	return js.nc.Drain()
 }
+
+// Handle nats signature request for challenge response authentication.
+func sigHandler(seedKey string, nonce []byte) ([]byte, error) {
+	kp, err := nkeys.FromSeed([]byte(seedKey))
+	if err != nil {
+		return nil, err
+	}
+	// Wipe our key on exit.
+	defer kp.Wipe()
+
+	sig, _ := kp.Sign(nonce)
```

Comment:
Skip error here is safe?

Reply:
Its basically a copy from https://github.com/nats-io/nats.go/blob/main/nats.go

But a non valid seed key should return an error on line 182 and when the signature is wrong or empty on line 189 this will result in a valid verification.